reference/orders.v2.oas2.yml

```diff
@@ -4720,5 +5029,365 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/shippingAddress_Base'
+            consignments:
+              $ref: '#/components/schemas/orderConsignment_Post'
         - $ref: '#/components/schemas/order_Shared'
       x-internal: false
+    orderConsignment_Get:
+      title: orderConsignment_Get
+      type: object
+      properties:
+        shipping:
+          type: array
+          items:
+            $ref: '#/components/schemas/shippingConsignment_Get'
+        downloads:
+          type: array
+          items:
+            $ref: '#/components/schemas/digitalConsignment_Get'
+          maxItems: 1
+        email:
+          type: object
+          properties:
+            gift_certificates:
+              type: array
+              items:
+                $ref: '#/components/schemas/giftCertificateConsignment_Get'
+    digitalConsignment_Post:
+      allOf:
+        - type: object
+          properties:
+            line_items:
+              required: true
+              type: array
+              minItems: 1
+              items:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/orderCatalogProduct_Post'
+      x-examples:
+        x-example-1:
+          line_items:
+            - url: https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2/orders/129/products/12
+              resource: /orders/129/products/12
+    shippingConsignment_Post:
+      allOf:
+        - $ref: '#/components/schemas/shippingConsignment_Base'
+        - type: object
+          properties:
+            line_items:
+              type: array
+              minItems: 1
+              items:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/orderCustomProduct_Post'
+                    - $ref: '#/components/schemas/orderCatalogProduct_Post'
+          required:
+            - line_items
```

Comment:
I missed a couple of required fields in my original PR 
- `zip`
- either `country` or `country_iso2`

Reply:
Thanks for that info @WillemHoman